### DISCUSSION TITLE
How to make IdeaVim choose the way to show line number?

### DISCUSSION THREAD

**Discussion by rainbowechoes** [JetBrains/ideavim#851] [Q&A]
Hi, community

I want to make IdeaVim show the absolute line number when I'm in normal mode, and then show the relative line number when I'm in insert mode.

I tried to add these configs in .vimrc, but it didn't work. What should I do? Anyone can help me?

```
augroup numbertoggle
  autocmd!
  autocmd BufEnter,FocusGained,InsertLeave,WinEnter * if &nu && mode() != "i" | set rnu   | endif
  autocmd BufLeave,FocusLost,InsertEnter,WinLeave   * if &nu                  | set nornu | endif
augroup END

```

**Comment by AlexPl292** [comment]
Hi there. Unfortunatelly, `autocmd` is not yet supported: [VIM-1693](https://youtrack.jetbrains.com/issue/VIM-1693) Add support for autocmd
